webpack.config.js

```diff
@@ -97,17 +99,17 @@ module.exports = [
             modules: ['./static', "./node_modules"],
             alias: {
                 //is this safe?
-                goldenlayout:  path.resolve(__dirname, 'node_modules/golden-layout/'),
-                lzstring:  path.resolve(__dirname, 'node_modules/lz-string/'),
-                filesaver:  path.resolve(__dirname, 'node_modules/file-saver/'),
-                vs: path.resolve(__dirname, 'node_modules/monaco-editor/min/vs')
+                goldenlayout: path.resolve(__dirname, 'node_modules/golden-layout/'),
+                lzstring: path.resolve(__dirname, 'node_modules/lz-string/'),
+                filesaver: path.resolve(__dirname, 'node_modules/file-saver/')
             }
         },
-        stats: "errors-only",
+        stats: "normal",
         devtool: 'source-map',
         optimization: {
             minimize: !isDev,
             minimizer: [new UglifyJsPlugin({
+                exclude: /\/vs/,
```

Comment:
should this not be `/\/monaco-editor/`?

Reply:
Good catch. It's not needed anymore though. I still needed to remove the /vs/ output path